### PR TITLE
Update Material Design Icons to v5.6.55

### PR DESCRIPTION
**Pull request by mririgoyen**
## Breaking change

The `scooter` icon is introduced in this release. This icon replaces the icon that is now renamed to `scooter-human` and was marked to be removed in release 0.117.

## Proposed change

Update Material Design Icons to version 5.6.55. See the [changelog here](http://dev.materialdesignicons.com/changelog#version-5.6.55).

## Type of change

- [x] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

Material Design Icons changed their license in the v5.6.55 release. The new license is available here: http://dev.materialdesignicons.com/license

The TL;DR is that MDI icons are now released under Apache 2.0, whereas before they were released partially under Apache 2.0 and MIT. This should not affect Home Assistant.

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by ludeeus** [comment]
Note: This version adds back a `scooter`  icon, which we are currently warning about until 0.117.
<https://github.com/home-assistant/frontend/blob/dev/src/components/ha-icon.ts#L35>

**Comment by bramkragten** [comment]
So we should wait with updating to 5.6.55 till 0.117?

**Comment by ludeeus** [comment]
Not really sure what's the best approach here.
The cleanest way would be to add this and remove the warning we have now and make sure we add a note about that in the 0.116 release notes.

**Comment by goyney** [comment]
v5.6.55 does add quite a few Home-specific icons: http://materialdesignicons.com/cdn/5.6.55/

I think I agree with @ludeeus' first approach. The difference between `scooter` and `human-scooter` is a person, so the meaning isn't completely lost if it does affect anyone, so a simple Breaking Changes note should suffice.

**Review by bramkragten — APPROVED** [review]
